File: src/main/java/net/slipp/repository/smalltalk/SmallTalkCommentRepository.java
package net.slipp.repository.smalltalk;

import java.util.List;

import net.slipp.domain.smalltalk.SmallTalk;
import net.slipp.domain.smalltalk.SmallTalkComment;
import net.slipp.domain.user.SocialUser;

import org.springframework.data.repository.CrudRepository;

public interface SmallTalkCommentRepository extends CrudRepository<SmallTalkComment, Long> {
	List<SmallTalkComment> findBySmallTalk(SmallTalk smallTalk);

	List<SmallTalkComment> findByWriter(SocialUser writer);
}

File: src/main/scala/net/slipp/repository/smalltalk/SmallTalkCommentRepository.scala
package net.slipp.repository.smalltalk

import java.lang.Long
import java.util.List
import net.slipp.domain.smalltalk.SmallTalkComment
import net.slipp.domain.smalltalk.SmallTalk
import net.slipp.domain.user.SocialUser
import org.springframework.data.repository.CrudRepository

trait SmallTalkCommentRepository extends CrudRepository[SmallTalkComment, Long] {
  def findBySmallTalk(smallTalk: SmallTalk): List[SmallTalkComment]

  def findByWriter(writer: SocialUser): List[SmallTalkComment]
}